frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -275,6 +279,25 @@ describe("ValuesSourceModal", () => {
     });
 
     it("should allow searching for a card without access to the root collection (metabase#30355)", async () => {
+      fetchMock.get(
```

Comment:
When navigating to the search UI there are a number of API calls made that weren't causing things to fail before, but are now.